psi4/driver/driver_cbs.py

```diff
@@ -67,16 +71,16 @@ def _expand_bracketed_basis(basisstring: str, molecule=None):
         for sensible sequence and returned as separate basis sets
         (e.g., ``'cc-pV[Q5]Z'` is returned as ``(["cc-pVQZ", "cc-pV5Z"], [4, 5])``).
         Allows out-of-order zeta specification (e.g., ``[qtd]``) and numeral for
-        number (e.g., ``[23]``). Does not allow skipped zetas (e.g., ``[dq]``), zetas
-        outside the [2,8] range, non-Dunning, non-Ahlrichs, or non-Jensen sets,
-        or non-findable .gbs sets.
+        number (e.g., ``[23]``). Does not allow skipped zetas (e.g., ``[dq]``), 
+        zetas outside the [2,8] range, non-Dunning, non-Ahlrichs, or non-Jensen 
+        sets, or non-findable .gbs sets.
     molecule : qcdb.molecule or psi4.core.Molecule
         This function checks that the basis is valid by trying to build
         the qcdb.BasisSet object for *molecule* or for H2 if None.
 
     Returns
     -------
-    tuple
+    (BSET, ZSET) : Tuple[List[str], List[int]]
```

Comment:
Isn't it redundant to have the type information here, now that the return type is annotated? I have this comment many times over.

Reply:
Yes in principle, but Sphinx is not yet clever enough to read the return annotations from the header.